imagelab/DisplayImage.java

```diff
@@ -14,78 +16,92 @@
     /** Serialization version. */
     private static final long serialVersionUID = 11L;
     /** The image of this frame. */
-    Image img;
+    private Image img;
     /** The display panel of this frame. */
     private DisPanel pane;
 
-    /** Initial x-coordinate for window placement */
+    /** Initial x-coordinate for window placement. */
     private static final int XINIT = 10;
-    /** Initial y-coordinate for window placement */
+    /** Initial y-coordinate for window placement. */
     private static final int YINIT = 20;
-    /** x-coordinate delta for subsequent window */
+    /** x-coordinate delta for subsequent window. */
     private static final int XDELTA = 25;
-    /** y-coordinate delta for subsequent window */
+    /** y-coordinate delta for subsequent window. */
     private static final int YDELTA = 25;
-    /** Maximum x-coordinate before wrapping */
+    /** Maximum x-coordinate before wrapping. */
     private static final int XMAX = 800;
-    /** Maximum y-coordinate before wrapping */
+    /** Maximum y-coordinate before wrapping. */
     private static final int YMAX = 600;
-    
-    /** x-coordinate for next window */
+    /**DisplayImage background color. */
+    private static final int COLOR = 255;
```

Comment:
Not a meaningful comment.  Why is the value of COLOR equal to 255?  Need an identifier that is more informative.